frontend/src/components/button/button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+interface IProps {
+  children: React.ReactNode;
+  href: string;
+  accentColor?: "pink" | "red" | "green" | "blue";
+}
+
+const Button = ({
+  children,
+  href,
+  accentColor = "pink",
+}: IProps): JSX.Element => {
+  return (
+    <span className="inline-flex rounded-md shadow-sm">
+      <Link
```

Comment:
Mayde there should be some logic to check if button is functional (trigger some function) or link and based on that use `Link` or button in HTML but fine for now